.github/workflows/ci_checks.yml

```diff
@@ -163,7 +142,7 @@ jobs:
 
   qa_purge_env:
     runs-on: [ubuntu-latest]
-    needs: [qa_lint, qa_unit_tests, mobius_migration_tests, qa_android_tests]
```

Comment:
We still want `qa-android-tests` as required, right? Why are we removing it from here?

Reply:
We are, if you look at `ci_checks_pr_comment.yml`, there is another `qa_purge_env` job that needs `qa_android_tests`. We removed it from here, because it's not longer part of this GH Action.